docs/root/faq/disable_circuit_breaking.rst

```diff
@@ -0,0 +1,7 @@
+Is there a way to disable circuit breaking?
+===========================================
+
+Envoy comes with :ref:`certain defaults <envoy_api_msg_cluster.CircuitBreakers.Thresholds>`
+for each kind of circuit breaking. Currently, there isn't a switch to turn
+circuit breaking off completely; however, you could achieve a similar behavior
+by setting these thresholds very high, for example, to `std::numeric_limits<uint32_t>::max()`.
```

Comment:
If you are up for it could you add a config snippet as an example? That is very useful for folks.

Reply:
Addressed in 25a8cce.